src/main/java/com/gamebros/purepazaak/MatchSet.java

```diff
@@ -0,0 +1,118 @@
+package com.gamebros.purepazaak;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import com.gamebros.purepazaak.entity.Card;
+import com.gamebros.purepazaak.entity.Player;
+import com.gamebros.purepazaak.enumerable.PlayerEnum;
+import com.gamebros.purepazaak.enumerable.WinnerEnum;
+
+public class MatchSet {
+  protected Board playerOneBoard = new Board();
+
+  protected Board playerTwoBoard = new Board();
+
+  protected MainDeck deck;
+
+  protected WinnerEnum winner = WinnerEnum.NONE;
+
+  protected PlayerEnum currentPlayer = PlayerEnum.PLAYERONE;
+
+  protected HashSet<PlayerEnum> playersStood = new HashSet<PlayerEnum>();
+
+  public MatchSet(MainDeck deck) {
+    this.deck = deck;
+
+    this.draw();
```

Comment:
This call to `draw` shouldn't happen within the constructor.  It should be the job of the Controller.